gradle-baseline-java/src/test/groovy/com/palantir/baseline/BaselineExactDependenciesTest.groovy

```diff
@@ -29,7 +31,7 @@ class BaselineExactDependenciesTest extends AbstractPluginTest {
             id 'java'
             id 'com.palantir.baseline-exact-dependencies'
             id 'com.palantir.baseline' apply false
-            id 'com.palantir.consistent-versions' version '2.0.0' apply false
+            id 'com.palantir.consistent-versions' version '2.31.0' apply false
```

Comment:
not sure what to do with this format of declaration yet, so just bumping the version manually for now.